about.html

```diff
@@ -0,0 +1,30 @@
+<html>
+
+  <head>
```

Comment:
Hi! Nice to meet you! My name is Estefanía and I'll be taking a look at your code today. 👋😃